### ISSUE TITLE
Access modifier inspection incorrectly suggests class constants be declared public

### ISSUE THREAD

**Issue by BinaryKitten** [kalessil/phpinspectionsea#899]
PhpStorm 2018.1.1
Build #PS-181.4445.72, built on April 9, 2018
JRE: 1.8.0_152-release-1136-b27 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

PHP Inspections Ultimate Version 2.0.2
![phpinspections_version](https://user-images.githubusercontent.com/67553/39304081-5ea86fce-4950-11e8-91d7-c8b279f4c636.png)

See image code:
![phpstorm64_2018-04-26_12-40-11](https://user-images.githubusercontent.com/67553/39303917-cd84466c-494f-11e8-8fd9-7f0eb8b816eb.png)

```php
<?php

use Illuminate\Database\Eloquent\Model as EloquentModel;

class Categories extends EloquentModel
{
    const CATEGORY_SECTION_BLOG   = 0;
    const CATEGORY_SECTION_NEWS   = 1;
    const CATEGORY_SECTION_EVENTS = 2;

    public    $timestamps = false;
    protected $table      = 'categories';
}
```

**Comment by funivan** [comment]
This inspection suggests adding visibility modifier to the property for example `public` or `private`.
Why do you think this inspection is incorrect?

**Comment by BinaryKitten** [comment]
because Constants don't have a visibility setting. If you add Public before Const you get a fatal error of unexpected T_CONST

**Comment by funivan** [comment]
http://php.net/manual/en/language.oop5.constants.php
```
Note:
As of PHP 7.1.0 visibility modifiers are allowed for class constants.
```
Please check your php version specified in the project. 
Go to `language and frameworks` -> `php`

![03 03 43](https://user-images.githubusercontent.com/425208/39304682-1f56b55c-4963-11e8-9c49-25691041db50.png)

**Comment by funivan** [comment]
You can specify 7.0 and then this inspcetion should not warn you about constant visibility.

**Comment by BinaryKitten** [comment]
My error. 
Sorry about that. Seems someone had modified the dev server away from 7.1 to 7.0. Thanks for pointing this out.

**Comment by funivan** [comment]
No problem. Glad to help.
